demo/EV3D4/README.md

```diff
@@ -1,21 +1,20 @@
 # EV3D4
 EV3D4 is designed to look like R2-D2 from Star Wars. There are two options for
 controlling EV3D4. The first is to use the IR remote to send commands to the IR
-sensor, run EV3D4RemoteControl to use this method. The second means of
-controlling EV3D4 is via a web browser, run EV3D4WebControl to use this method.
+sensor, run EV3D4RemoteControl.py to use this method. The second means of
+controlling EV3D4 is via a web browser, run EV3D4WebControl.py to use this method.
+You can run both of these from the Brickman interface or if logged in via ssh
```

Comment:
I will go ahead and merge this one (the moving of the web server code from helper.py to webserver.py seems ok) and do a separate pull request that addresses:
- How do I install the demo programs on my EV3?
- How do I run the demo programs on my EV3?